src/main/asciidoc/main.adoc

```diff
@@ -1 +1,50 @@
-= Test API guide
+= Nutaxi Route Service API
+
+:doctype: book
+:icons: font
+:source-highlighter: highlightjs
+:sectlinks:
+
+
+[[overview]]
+= Overview
+
+Route service API overview
+
+
+[[resources]]
+= Resources
+
+[[route]]
+== Route
+
+[[route-desc]]
+=== Getting a route
+
+A `GET` request will list all of the service's notes.
+
+[[route-request]]
+==== Request structure
+
+include::{snippets}/route/http-request.adoc[]
+
+[[route-request-parameters]]
+==== Parameters
```

Comment:
Because there is no locale parameter, added a note that `Accept-Language` HTTP header may have affect on the response in the future.